backend/apis/models.py

```diff
@@ -17,7 +17,6 @@ class User(UserMixin, db.Model):
     location = db.Column(db.String(120))
```

Comment:
instead of hardcoding this `120`, how about we make this a global variable to give it meaning. for instance, make it a variable called `DB_STRING_LENGTH = 120`

Reply:
IMO, for scalability purposes, we should have another file in the project called `constants`, which stores all the variables to prevent any hardcoding @adialachar